tests/mock_callable_testslide.py

```diff
@@ -193,7 +211,14 @@ def it_rejects_unknown_arguments(self):
                                             self.specific_call_kwargs.keys()
                                         )
                                     )
-                                    + "    }\n",
+                                    + "    }\n"
+                                )
+                                rgx = (
+                                    "The mock_callable configured at ((?:[^/]*/)*)(.*):\d{0,4} encountered unexpected behavior:\n"
```

Comment:
I believe we can get significant more coverage here if we use [``__file__``](https://docs.python.org/3/reference/import.html#__file__) instead of `((?:[^/]*/)*)(.*)`. Making this match the line on a case by case would be a bit of a pain to keep up to date, I wouldn't bother with that.